src/partest-extras/scala/tools/partest/ASMConverters.scala

```diff
@@ -119,6 +123,13 @@ object ASMConverters {
       case i: t.LineNumberNode         => LineNumber   (i.line, applyLabel(i.start))
     }
 
+    private def convertBsmArgs(a: Array[Object]): List[Object] = a.map({
```

Comment:
I left this as is because the second parameter list `(collection.breakOut)` looks strange otherwise.